### PR TITLE
OSSM-4203: Add prow jobs for maistra/xns-infromer 2.5

### PR DESCRIPTION
**Pull request by jewertow**

### CODE REVIEW

**Review by jwendell — CHANGES_REQUESTED** [review]
Change the files only in the `ci-operator/config` directory and run `make update`. Commit everything.

File: ci-operator/config/maistra/xns-informer/maistra-xns-informer-maistra-2.5.yaml [hunk 1]
```diff
@@ -0,0 +1,40 @@
+build_root:
+  image_stream_tag:
+    name: maistra-builder
+    namespace: ci
+    tag: "2.5"
```
Comment (jwendell) [inline ci-operator/config/maistra/xns-informer/maistra-xns-informer-maistra-2.5.yaml:5]: This will fail, as we don't have a builder image for 2.5 yet.

Comment (jewertow) [reply ci-operator/config/maistra/xns-informer/maistra-xns-informer-maistra-2.5.yaml:5]: `quay.io/maistra-dev/maistra-builder:2.5` is already available

**Review by jwendell — APPROVED** [review]
The linter failure can be fixed in the repo

**Pull request merged**